### PR TITLE
docs: document http-terminator (fixes #1096)

### LINKED ISSUE #1096: Add http-terminator to Graceful Shutdown documentation
One of the most often support questions I see in IRC is how to terminate express.js server/ why does server not stop when calling `server.close()`. I have written a library that handles express.js termination with a graceful timeout for the ongoing requests.

https://github.com/gajus/http-terminator

If Okay expressjs.com maintainers, I would like to add it to https://expressjs.com/en/advanced/healthcheck-graceful-shutdown.html to list of libraries.

### PR DESCRIPTION
**Pull request by gajus**

### CODE REVIEW

**Review by jonchurch — APPROVED** [review]
Edit: I've requested one change, but otherwise...
LGTM 👍 
This package actually looks quite good too btw, kudos! 

~~P.S. the failing CI doesn't have anything to do with this PR.~~

The CI was failing before this PR, but there is one styling error introduced by it, which I suggested a change on. ❤️

File: en/advanced/healthcheck-graceful-shutdown.md
Comment (jonchurch) [inline en/advanced/healthcheck-graceful-shutdown.md]:
```suggestion
  httpTerminator.terminate()
```
Actually this one extra semicolon is throwing an error in the CI for code styling

**Review by jonchurch — CHANGES_REQUESTED** [review]

**Review by jonchurch — APPROVED** [review]
Thanks! LGTM

**Comment by gajus** [comment]
Do I need to do anything regarding the failing tests?

**Comment by jonchurch** [comment]
@gajus Nope, that's not a blocker and there's a PR #1094 open that will fix the failing CI. 

This is pending review from @expressjs/expressjs-com and after that it will likely get merged in.

**Comment by wesleytodd** [comment]
This is not a judgement on the package, but I would like to see the express documentation have less documentation on third party packages.  Today there is a ton of outdated stuff which was added years ago, and I would like to not make it any worse.  I think we should consider moving stuff like this into an "awesome list" format instead of it being on the express website.  Thoughts about this direction @jonchurch?

**Comment by gajus** [comment]
> I think we should consider moving stuff like this into an "awesome list" format instead of it being on the express website.

What makes that different from the current format?

**Comment by wesleytodd** [comment]
We wouldn't maintain it :)  The problem with it on the website is it makes us already limited maintainers have more work.  In an awesome list I, and I am guessing the other primary maintainers would not maintain it.

**Comment by jonchurch** [comment]
I think having a discussion about cleaning up the site would be fruitful. IMO the site should document the project and not much else. 

However, as things stand we do have these kind of sections and I think OP’s library is high quality so Im pro merging. 

These kinds of sections (recommendations of libraries) can always be removed in the future, and I think it would be pragmatic to move towards that.

**Review by dougwilson — APPROVED** [review]
I know there was a bunch of conversation in this PR revolved around the greater idea on 3-rd party modules, but they are there today, and this one is really nice, so I'm going to merge it in and if there is ever a policy in the future, it's easy to adjust the contents at that time 👍

**Comment by gajus** [comment]
Thank you!

**Pull request merged**
